Gemfile.jruby-1.9.lock

```diff
@@ -132,8 +132,8 @@ GEM
     logstash-codec-graphite (2.0.2)
       logstash-codec-line
       logstash-core (>= 2.0.0.beta2, < 3.0.0)
-    logstash-codec-json (2.1.0)
-      logstash-core (>= 2.0.0.beta2, < 3.0.0)
+    logstash-codec-json (2.1.1)
+      logstash-core (>= 2.0.0, < 3.0.0)
```

Comment:
ok, so, we need to followup with an issue to make sure the gemspecs are consistent and that they should not be changed during a normal dev workflow, so this should have been caught in the PR review too. /cc @andrewvc since you made that commit

I'm good to move on if that new constrain does not  break things